.env

```diff
@@ -0,0 +1 @@
+GITHUB_TOKEN=b263368792c75b349311a742f91c21b5d8691c61
```

Comment:
Guessing you might want to revoke this token

Reply:
llolll thxxx revoking XD

Reply:
I knew that would happen XD I have a PR to .gitignore that file